package.json

```diff
@@ -232,6 +234,7 @@
     "THIRD_PARTY_NOTICES.md",
     "lib/",
     "bin/tracetractor",
-    "bin/test-naming-rules.js"
+    "bin/test-naming-rules.js",
+    "esm-loader.mjs"
```

Comment:
nice catch. this would've been a downer to get through the work and forget to include this here.